src/anim/binder/default-anim-binder.js

```diff
@@ -138,10 +138,14 @@ class DefaultAnimBinder {
 
             // #ifdef DEBUG
             var fallbackGraphPath = AnimBinder.encode(path.entityPath[path.entityPath.length - 1] || "", 'graph', path.propertyPath);
-            if (this.visitedFallbackGraphPaths[fallbackGraphPath]) {
+            if (this.visitedFallbackGraphPaths[fallbackGraphPath] === 1) {
                 console.warn('Anim Binder: Multiple animation curves with the path ' + fallbackGraphPath + ' are present in the ' + this.graph.path + ' graph which may result in the incorrect binding of animations');
             }
-            this.visitedFallbackGraphPaths[fallbackGraphPath] = true;
+            if (!Number.isFinite(this.visitedFallbackGraphPaths[fallbackGraphPath])) {
```

Comment:
this feels odd .. I assume visitedFallbackGraphPaths is private, so why do you need to test for invalid numbers? Should the code store valid numbers only instead?

Reply:
It only stores non negative numbers but there may not be an entry for a given path in the dictionary yet.

Reply:
there must be a way to just test if this property exists? Number.isFinite sounds like it tests more than needed and would cost more.

Reply:
It's because 0 is a falsy value so I wouldn't want to just test if a number exists.